shaky/src/main/java/com/linkedin/android/shaky/Shaky.java

```diff
@@ -186,12 +188,23 @@ public void hearShake() {
         arguments.putInt(ShakySettingDialog.SHAKY_CURRENT_SENSITIVITY, delegate.getSensitivityLevel());
         SendFeedbackDialog sendFeedbackDialog = new SendFeedbackDialog();
         sendFeedbackDialog.setArguments(arguments);
-        sendFeedbackDialog.show(activity.getFragmentManager(), SEND_FEEDBACK_TAG);
+        if (customFragment != null) {
+            customFragment.show(activity.getFragmentManager(), CUSTOM_DIALOG_TAG);
```

Comment:
tiny nit - it seems like a lot of the logic above this line could probably go in the `else` block since we're not actually doing anything with the `sendFeedbackDialog` right? And then similar question for the `shakyFlowCallback` on 171 and on 197.